### ISSUE TITLE
Metadata -> Convert latitude / longitude to geojson

### ISSUE THREAD

**Issue by halkeye** [jenkins-infra/stories#5]
https://decapcms.org/docs/widgets/#map
https://www.gatsbyjs.com/plugins/gatsby-transformer-geojson/

**Comment by biru-codeastromer** [comment]
@halkeye Sir ,
 the upper link is not working

**Comment by biru-codeastromer** [comment]
@krisstern Hello Sir,may you please let me know if this issue i can work if it is still existing!

**Comment by krisstern** [comment]
@biru-codeastromer We do not assign issues, but you can submit a PR for this issue if you understand the requirements. 

> @halkeye Sir , the upper link is not working

@biru-codeastromer I have updated the link now it should work

**Comment by biru-codeastromer** [comment]
sure Sir @krisstern Thanks very much for the update

**Comment by krisstern** [comment]
Are you still working on this @biru-codeastromer? We do not want to see any contributor hoarding issues to work on

**Comment by biru-codeastromer** [comment]
Hi Sir @krisstern,

I am still actively working on this issue. However, as per your earlier guidance, I was instructed to prioritize completing issue #1 before returning to this. I created a PR for issue #1, which you indicated was fine but required assistance from the infra team. Since the issues are closely interrelated, I was waiting for that to be resolved.

Could you please confirm if I can proceed with this issue independently of #1, or if I should wait for the resolution of the previous issue first?

Looking forward to your confirmation.
Thanks!


<img width="1512" alt="Screenshot 2025-01-15 at 9 47 21 AM" src="https://github.com/user-attachments/assets/93bd1a0a-4906-48ed-9ea1-4f0279b55a62" />

**Comment by krisstern** [comment]
I actually don't think you have a workable solution in #77, if you are working on #8 then I will close #77 so that anyone else interested can pick up on that issue.

**Comment by krisstern** [comment]
You will need to choose one of #5 or #8 to work on at any one time, not both.

**Comment by biru-codeastromer** [comment]
Hi Sir @krisstern,

Thank you for the clarification. I understand the need to focus on one issue at a time. Since I’ve been interested working on #8, I will prioritize completing it first. Once that’s done, I’ll return to #5 if it’s still open or available.

If you prefer, you can close #77, and I’ll ensure I wrap up my work on #8 efficiently before moving forward.

Thank you for your guidance!

**Comment by biru-codeastromer** [comment]
> I actually don't think you have a workable solution in [#77](https://github.com/jenkins-infra/stories/pull/77), if you are working on [#8](https://github.com/jenkins-infra/stories/issues/8) then I will close [#77](https://github.com/jenkins-infra/stories/pull/77) so that anyone else interested can pick up on that issue.




Hi Sir , I wanted to ask you if my PR #84 get merged , Can I work on this Issue again after that . I will find a new solution and first inform you here about the approach .
I am asking this only after #84 get merged and if no one still working on this issue . Let me know if this is fine with you Sir.
Thanks !

**Comment by krisstern** [comment]
Sure @biru-codeastromer, you are more than welcome to open a new PR for this issue

**Comment by biru-codeastromer** [comment]
> Sure [@biru-codeastromer](https://github.com/biru-codeastromer), you are more than welcome to open a new PR for this issue

Thanks Sir !

**Comment by biru-codeastromer** [comment]
> Sure [@biru-codeastromer](https://github.com/biru-codeastromer), you are more than welcome to open a new PR for this issue

Hi Sir,

I’ve been working on updating the map field in YAML files to align with the Decap CMS map widget's requirement for GeoJSON strings in WGS84 projection.

My approach involves converting the latitude and longitude values in the map field into a GeoJSON object and embedding it directly in the YAML files. This would make the data fully compatible with the map widget and eliminate the need for additional transformations during usage.

Would this approach work for the intended integration with the CMS? I’d appreciate your feedback or any suggestions you might have!

If you see it as good one , I will raise the PR or should I raise the PR first.

Thanks!

**Comment by biru-codeastromer** [comment]
Hi Sir, I have raised a new PR #93 with a new approach !
